README.md

```diff
@@ -59,6 +59,7 @@ To add a resource or contribute, please check out our contribution guide [here](
   - [Open Source](#open-source)
   - [Standard Template Library](#standard-template-library)
   - [Programming Courses Free Online](#programming-courses-free-online)
+  - [Web Design & Development Toolkit](#web-design--development-toolkit)
```

Comment:
Instead of creating a new topic for this resource, please add it under Design which is under Other